content/blog/cc-product-roadmap-1/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Product Roadmap
+date: 2019-11-22
+tags: 
+  - product
+  - plans
+  - announcements
+image: release-1.jpg
+author: crysfel-villa
+---
+Over the last couple weeks we've been working on the new features we want to implement in the platform. We've finally came up with a plan for the improvements we will be working over the next couple months.
+
+While this is not set on stone, it'll definately give us the direction we need to keep this project moving forward. I'm really excited about the new things to come and I hope all of you can join us in this adventure.
+
+One of the goals is to deliver features as quickly as we can, for that reason we are going to release all new features incrementally. We are planning to have four small/medium releases over the next several months.
+
+### Release 1: User profiles
+At the moment mentees can't really do anything in the plaform, but it would be great if any user can update their profile so mentors can see their goals and expectations when a mentee reach out to them.
```

Comment:
```suggestion
At the moment mentees can't really do anything in the platform, but it would be great if any user can update their profile so mentors can see their goals and expectations when a mentee reaches out to them.
```